pandas/core/groupby/groupby.py

```diff
@@ -902,6 +905,15 @@ def __getattr__(self, attr: str):
             f"'{type(self).__name__}' object has no attribute '{attr}'"
         )
 
+    def __getattribute__(self, attr):
```

Comment:
can you type args & outputs

Reply:
Typed the arg, but the output has no type